api/src/test/java/org/openmrs/api/VisitServiceTest.java

```diff
@@ -955,6 +980,34 @@ public void saveVisit_shouldNotPersistNewEncounter() {
 		Context.flushSession();
 	}
 
+	/**
+	 * @see VisitService#purgeVisit(Visit) 
+	 */
+	@Test
+	public void purgeVisit_shouldPurge() {
+		Visit visit = visitService.getVisit(1);
+		assertNotNull(visit);
+
+		visitService.purgeVisit(visit);
+		visit = visitService.getVisit(1);
+		assertNull(visit);
+	}
+
+	/**
+	 * @see VisitService#purgeVisit(Visit)
+	 */
+	@Test(expected = APIException.class)
```

Comment:
please remove this test as its already tested in `purgeVisit_shouldFailIfTheVisitHasEncountersAssociatedToIt`